tests/components/sensor/test_recorder.py

```diff
@@ -82,11 +82,11 @@ def set_time_zone():
         ("battery", None, None, 13.050847, -10, 30),
         ("humidity", "%", "%", 13.050847, -10, 30),
         ("humidity", None, None, 13.050847, -10, 30),
-        ("pressure", "Pa", "Pa", 13.050847, -10, 30),
-        ("pressure", "hPa", "Pa", 1305.0847, -1000, 3000),
-        ("pressure", "mbar", "Pa", 1305.0847, -1000, 3000),
-        ("pressure", "inHg", "Pa", 44195.25, -33863.89, 101591.67),
-        ("pressure", "psi", "Pa", 89982.42, -68947.57, 206842.71),
+        ("pressure", "Pa", "hPa", 0.13050847, -0.1, 0.3),
```

Comment:
The units used when creating long term statistics pressure sensor is specified here:
https://github.com/home-assistant/core/blob/986b60e527c02d42ce44f3bc9d98cb43b7e54dec/homeassistant/components/sensor/recorder.py#L93

Does the test fail if it's not updated?